### PR TITLE
Add an additional toolshed guide with examples.

### PR DESCRIPTION
**Pull request by dffdff2423**
This PR includes some minor alterations to the introduction to toolshead and
adds a ton of examples for how to use toolshead. The examples are all from
"Milon's Guide to Toolshed" which has been privately shared for some time. The
content was published with permission from the author. It was originally
targeted at game admins, however should also be useful to developers.

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Robust Toolbox
     - [Entities](en/robust-toolbox/toolshed/commands/entity-control.md)
     - [General](en/robust-toolbox/toolshed/commands/general.md)
     - [Miscellaneous](en/robust-toolbox/toolshed/commands/misc.md)
+  - [Tollshead Examples](en/robust-toolbox/toolshed/toolshead-examples.md)
```
Comment (Partmedia) [inline src/SUMMARY.md:80]: "Toolshed" is the preferred spelling

Comment (Partmedia) [reply src/SUMMARY.md:80]: Check the filename, too

File: src/en/robust-toolbox/toolshed.md [hunk 2]
```diff
@@ -32,3 +32,9 @@ Toolshed also supports variables in which you can store values. You can use the
 ```
 {{#include toolshed/explain_example_3.txt}}
 ```
+
+```admonish note
+Toolshead often spits out lengthy stacktraces upon a command being used incorrectly. Typically, there is a more clear error above the stacktrace in your console.
```
Comment (Partmedia) [inline src/en/robust-toolbox/toolshed.md:37]: Spelling

File: src/en/robust-toolbox/toolshed/toolshead-examples.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
+# Toolshead Examples
```
Comment (Partmedia) [inline src/en/robust-toolbox/toolshed/toolshead-examples.md:1]: Spelling

**Review by Partmedia — COMMENTED** [review]
Thanks for the contribution. I think this is long overdue and this will help a lot of people. I have some minor nitpicks.

File: src/SUMMARY.md (hunk 1 above)
Comment (dffdff2423) [reply src/SUMMARY.md:80]: Oh god, I see it now. I hate my dyslexia.

**Comment by dffdff2423** [comment]
Let me know if I missed something.

File: src/en/robust-toolbox/toolshed/toolshed-examples.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
+# Toolshed Examples
+
+This guide covers some basic examples of toolshead commands.
```
Comment (0x6273) [inline src/en/robust-toolbox/toolshed/toolshed-examples.md:3]:
```suggestion
This guide covers some basic examples of Toolshed commands.
```

**Review by 0x6273 — CHANGES_REQUESTED** [review]

File: src/en/robust-toolbox/toolshed/toolshed-examples.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+Make sure to replace the 1000 with the balance you want to set
+Alternatively, you could go into F7 > Objects > Stations, check the stations ENTID and then do
+```
+vv /entity/ENTID/StationBankAccount
+```
+
+## Scripts and You
+
+Scripts can be used to automate certain tasks or commands. To create a new script, you will need to create a new file in your Space Station 14 Data directory. On Windows you should be able to find it under %appdata%, in Space Staion 14/data
```
[the remark this answers is not in the document]
Comment (dffdff2423) [reply src/en/robust-toolbox/toolshed/toolshed-examples.md:172]: I don't think we should opt for linux over windows here. I would wager most linux users would know where to look for this without needing the documentation where as windows users might need the reminder. Also this is documented elsewhere.

File: src/en/robust-toolbox/toolshed/toolshed-examples.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+
+- We start off by getting our current character with `self`
+- Then for that character we execute the command `spawn:on` 10 times: `rep 10 spawn:on "MobAngryBee"`
+    - The `rep 10` part repeats the command ten times, and the `spawn:on` command will spawn a provided prototype MobAngryBee on the provided entity - here that's self
+- Then we want to fetch everything in a ten tile radius and filter off the bees we spawned with `nearby 10 prototyped MobAngryBee`
+- Once we have all of our bees, we select one of them using `select 1` and add the tag `DoorBumpOpener` using the command `tag:add`
+    - Tags are like components that contain no data, `DoorBumpOpener` allows for opening airlocks by walking into them, just like a player would
+- Next, for the same one bee, we execute the legacy command `makeghostrole` - the first argument is the ID, the second one is the name, the third one is the description. It's important to use the \ symbol before the quotation mark to ensure it is escaped properly.
+- Finally, we rename our bee to "Queen Bee"
+- Then we fetch all nine other bees and also turn them into ghost roles, just like we did with the Queen Bee
```
[the remark this answers is not in the document]
Comment (dffdff2423) [reply src/en/robust-toolbox/toolshed/toolshed-examples.md:137]: Will do (this is what I get from copy pasting a guide from discord)

File: src/en/robust-toolbox/toolshed/toolshed-examples.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (0x6273) [reply src/en/robust-toolbox/toolshed/toolshed-examples.md]: If it's documented elsewhere then make it a link.

[the remark this answers is not in the document]
Comment (dffdff2423) [reply src/en/robust-toolbox/toolshed/toolshed-examples.md]: Already did

**Comment by MilonPL** [comment]
I originally wrote this thing at 11 PM after three shots of espresso for the adminbus, so apologies for all the spelling mistakes. Glad to see it being PRed here!

**Comment by dffdff2423** [comment]
> I originally wrote this thing at 11 PM after three shots of espresso for the adminbus, so apologies for all the spelling mistakes. Glad to see it being PRed here!

No worries (half of them are probably mine from when I ported it). Sorry I did not realize you had a github, let me rebase this to add you as a co-author so you are credited in the commit log.

**Comment by dffdff2423** [comment]
Is there anything left for me to do? Or is it just waiting on reviews?

**Comment by Partmedia** [comment]
Sorry for the delay. There you go. Thanks for the contribution.

**Pull request merged**